components/crossplane-config/staging/provider-config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
+  name: eaas-stage-kubernetes-provider-config
+  api-server: https://api.kflux-stg-es01.21tc.p1.openshiftapps.com:6443
```

Comment:
This isn't a valid field and it's not needed anyway. The apiserver should be included as a key in the secret. Please remove this.